docs/ManualDeploymentGuideScalarDLOnEKS.md

```diff
@@ -1,74 +1,53 @@
-# Deploy Scalar DL Ledger on Amazon EKS (Amazon Elastic Kubernetes Service)
+# Deploy ScalarDL Ledger on Amazon EKS (Amazon Elastic Kubernetes Service)
 
-Scalar DL is scalable and practical Byzantine fault detection middleware for transactional database systems, which achieves correctness, scalability, and database agnosticism.  
+ScalarDL is scalable and practical Byzantine fault detection middleware for transactional database systems, which achieves correctness, scalability, and database agnosticism.
 
-Scalar DL is composed of [Ledger](https://github.com/scalar-labs/scalardl/blob/master/docs/getting-started.md), [Auditor](https://github.com/scalar-labs/scalardl/blob/master/docs/getting-started-auditor.md), and [Client SDK](https://github.com/scalar-labs/scalardl/tree/master/docs#client-sdks). Scalar DL Ledger manages application data in its own unique way using hash-chain and digital signature. Scalar DL Auditor is an optional component and manages a copy of Ledger data without depending on Ledger to identify the discrepancy between Ledger and Auditor data. The Client SDK is a set of user-facing programs to interact with Ledger and Auditor.  
+ScalarDL is composed of [Ledger](https://github.com/scalar-labs/scalardl/blob/master/docs/getting-started.md), [Auditor](https://github.com/scalar-labs/scalardl/blob/master/docs/getting-started-auditor.md), and [Client SDK](https://github.com/scalar-labs/scalardl/tree/master/docs#client-sdks). ScalarDL Ledger manages application data in its own unique way using hash-chain and digital signature. ScalarDL Auditor is an optional component and manages a copy of Ledger data without depending on Ledger to identify the discrepancy between Ledger and Auditor data. The Client SDK is a set of user-facing programs to interact with Ledger and Auditor.
 
-We can deploy **Scalar DL Ledger** on any Kubernetes services. This document explains how to deploy **Scalar DL Ledger** on EKS.  
+We can deploy **ScalarDL Ledger** on any Kubernetes services. This document explains how to deploy **ScalarDL Ledger** on EKS.
 
 ## What we create
 
-In this guide, we create the following environment on your AWS account.  
+In this guide, we create the following environment on your AWS account.
```

Comment:
Similar to my other comment in the other deployment guides, I feel like users should already know what ScalarDL Ledger is at this point. How about if we simplify the beginning of this document as follows?:

```suggestion
This document explains how to deploy **ScalarDL Ledger** on Amazon Elastic Kubernetes Service (EKS).

In this guide, you will create the following environment in your AWS environment account.
```